daemon/execdriver/native/create.go

```diff
@@ -222,3 +227,17 @@ func (d *driver) setupLabels(container *libcontainer.Config, c *execdriver.Comma
 
 	return nil
 }
+
+func (d *driver) setupAdditionalGroups(container *libcontainer.Config, c *execdriver.Command) error {
+	container.AdditionalGroups = make([]int, len(c.GroupAdd))
+
+	for i, g := range c.GroupAdd {
+		group, err := strconv.Atoi(g)
```

Comment:
If we accept group names - are they from the host's groups file or the
container's?  The latter is much more complicated, no?

On Wed, Feb 11, 2015 at 3:48 PM, Mrunal Patel <notifications@github.com>
wrote:

> In daemon/execdriver/native/create.go
> <https://github.com/docker/docker/pull/10717#discussion_r24546022>:
>
> > @@ -222,3 +227,17 @@ func (d *driver) setupLabels(container *libcontainer.Config, c *execdriver.Comma
> >
> >  	return nil
> >  }
> > +
> > +func (d *driver) setupAdditionalGroups(container *libcontainer.Config, c *execdriver.Command) error {
> > +	container.AdditionalGroups = make([]int, len(c.GroupAdd))
> > +
> > +	for i, g := range c.GroupAdd {
> > +		group, err := strconv.Atoi(g)
>
> @thockin <https://github.com/thockin> I expect review comments that we
> should probably accept group names. If not, then I can modify the code to
> use []int all the way up the stack.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/10717/files#r24546022>.
>